geopandas/base.py

```diff
@@ -94,7 +94,31 @@ class GeoPandasBase(object):
     @property
     def area(self):
         """Returns a ``Series`` containing the area of each geometry in the
-        ``GeoSeries``."""
+        ``GeoSeries``.
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Polygon
+        >>> s = geopandas.GeoSeries(
+        ...     [
+        ...         Polygon([(0, 0), (1, 1), (0, 1)]),
+        ...         Polygon([(10, 0), (10, 5), (0, 0)]),
```

Comment:
Suggestion: it might be good to add other geometry types to this example.  I.e., in this case, to demonstrate that LineString and Point both return an area of 0 (instead of raising an error)